pulp_python/app/serializers.py

```diff
@@ -22,6 +23,50 @@ class Meta:
         fields = ('name',)
 
 
+class DistributionDigestSerializer(serializers.ModelSerializer):
+    """
+    A serializer for the Distribution Digest in a Project Specifier
+    """
+    type = serializers.CharField(
+        help_text=_("A type of digest: i.e. sha256, md5")
+    )
+
+    digest = serializers.CharField(
+        help_text=_("The digest of the distribution")
+    )
+
+    class Meta:
+        model = python_models.DistributionDigest
+        fields = ('type', 'digest')
+
+
+class ProjectSpecifierSerializer(serializers.ModelSerializer):
+    """
+    A serializer for Python project specifiers
+    """
+
+    name = serializers.CharField(
+        help_text=_("A python project name.")
+    )
+
+    version_specifier = serializers.CharField(
+        help_text=_("A version specifier, accepts standard python versions "
+                    "syntax: >=, <=, ==, ~=, >, <, can be used in conjunction with"
+                    "multiple specifiers like "),
+        required=False,
+        allow_blank=True
+    )
+
+    digests = DistributionDigestSerializer(
```

Comment:
Same here I think, we can/should validate that the digest type is supported and the correct length for whatever type it is.